Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -57,66 +57,66 @@
     <!-- not explicitly defined, up to tracker design -->
     <!-- Tracker Forward -->
     <constant name="FwdTracker_id" value="4"/>
-    <constant name="FwdTracker_rmin1" value="60*mm"/>
-    <constant name="FwdTracker_rmax1" value="3500*mm"/>
-    <constant name="FwdTracker_z1" value="2500*mm"/>
-    <constant name="FwdTracker_rmin2" value="2000*mm"/>
-    <constant name="FwdTracker_rmax2" value="3505*mm"/>
-    <constant name="FwdTracker_z2" value="2000*mm"/>
-    <constant name="FwdTracker_zOffset" value="14000*mm"/>
-
-    <!-- E(M)CAL DIMENSIONS -->
-    <!-- E(M)Cal Barrel -->
+    <!-- single layer close to ExCal -->
+    <constant name="FwdTracker_rmin1" value="40*mm"/>
+    <constant name="FwdTracker_rmax1" value="1076.8*mm"/>
+    <constant name="FwdTracker_zOffset1" value="7500*mm"/>
+    <!-- the rest of the tracking layers inside the forward solenoid / shielding -->
+    <constant name="FwdTracker_rmin2" value="60*mm"/>
+    <constant name="FwdTracker_rmax2" value="1556*mm"/>
+    <constant name="FwdTracker_z2" value="3000*mm"/>
```

Comment:
If these values are a problem, I may fit the fwd tracker into volume between 10-16m. The average disc values are at 10 and 16m, respectively.